### PR TITLE
Site: add link to external tutorials based on minikube

### PR DESCRIPTION
**Pull request by RA489**
Site: add link to external tutorials based on minikube
fixes https://github.com/kubernetes/minikube/issues/5126

### CODE REVIEW

File: site/content/en/docs/Tutorials/external_tutorials.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+linkTitle: "External Tutorials On Minikube"
+weight: 1
+date: 2019-10-31
+description: >
+  Adding link to external tutorials based on minikube
+---
+
+Some of the good blogs posts and tutorials that are based on minikube are given below.
+
+Read more about minikube here: https://quarkus.io/guides/kubernetes-guide.html#deploying-the-application-in-kubernetes
```
Comment (afbjorklund) [inline site/content/en/docs/Tutorials/external_tutorials.md]: Obviously this place is now where to "read more about minikube", it is about Quarkus (a Java stack) ?

And it seems they moved the page, it is now here: https://quarkus.io/guides/deploying-to-kubernetes

File: site/content/en/docs/Tutorials/external_tutorials.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: "External Tutorials On Minikube"
+linkTitle: "External Tutorials On Minikube"
+weight: 1
+date: 2019-10-31
+description: >
+  Adding link to external tutorials based on minikube
+---
+
+Some of the good blogs posts and tutorials that are based on minikube are given below.
```
[the remark this answers is not in the document]
Comment (RA489) [reply site/content/en/docs/Tutorials/external_tutorials.md]: sure

File: site/content/en/docs/Tutorials/external_tutorials.md (hunk 1 above)
Comment (RA489) [reply site/content/en/docs/Tutorials/external_tutorials.md]: Yes they have moved the page here https://quarkus.io/guides/deploying-to-kubernetes

Comment (RA489) [reply site/content/en/docs/Tutorials/external_tutorials.md]: I will update asap

**Comment by RA489** [comment]
@medyagh @afbjorklund I have updated the PR as per your suggestions. PTAL.

File: site/content/en/docs/Tutorials/external_tutorials.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (RA489) [reply site/content/en/docs/Tutorials/external_tutorials.md]: done

File: site/content/en/docs/Tutorials/external_tutorials.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+linkTitle: "External Tutorials On Minikube"
+weight: 1
+date: 2019-10-31
+description: >
+  Adding link to external tutorials based on minikube
+---
+
+External tutorials about minikube:
+
+https://quarkus.io/guides/deploying-to-kubernetes
```
Comment (tstromberg) [inline site/content/en/docs/Tutorials/external_tutorials.md]: I'm OK with the concept, but this tutorial does not do enough with minikube to really call it a tutorial. It does not focus on minikube, and barely fits with the idea that a tutorials are  "complete worked examples made up of multiple tasks that guide the user through a relatively simple but realistic scenario".

Can you find another external example to use?

Comment (RA489) [reply site/content/en/docs/Tutorials/external_tutorials.md]: sure

**Comment by afbjorklund** [comment]
@RA489 : Can we close this PR ?

**Pull request closed without merging**